docs/shared/migration/migration-angular.md

```diff
@@ -276,29 +267,29 @@ ng test <app name>
 
 If you are using `Karma` for unit testing:
 
-- Copy the `karma.conf.js` file to your `apps/<app name>` folder.
-- Copy the `test.ts` file to your `apps/<app name>/src` folder.
-- Copy the `test` target in your `architect` configuration from your Angular CLI `angular.json` file into the `targets` configuration in the `apps/<app name>/project.json` file in your Nx workspace.
+- Copy the `karma.conf.js` file to the root folder.
+- Copy the `test.ts` file to your `src` folder.
+- Copy the `test` target in your `architect` configuration from your Angular CLI `angular.json` file into the `targets` configuration in the `project.json` file in your Nx workspace.
 - Update the `test` target file paths to be relative to `apps/<app name>`.
 
-```jsonc {% fileName="apps/<app name>/project.json" %}
+```jsonc {% fileName="project.json" %}
```

Comment:
It's a lot more 1:1 now.. I'd say it's not worth showing this example.

The one thing I would say to do is to change `builder => executor`...